msi/build/jenkins.wxs

```diff
@@ -439,7 +439,8 @@
       <Publish Dialog="InstallDirDlg" Control="ChangeFolder" Event="SpawnDialog" Value="BrowseDlg" Order="2">1</Publish>
 
 
-      <Publish Dialog="ServiceCredDlg" Control="Next" Event="NewDialog" Value="ServicePortDlg" Order="1">NOT Installed</Publish>
+      <Publish Dialog="ServiceCredDlg" Control="Next" Property="JENKINS_ROOT" Value="C:\ProgramData\Jenkins\" Order="1"><![CDATA[SERVICE_LOGON_TYPE="ServiceLocalSystem"]]></Publish>
```

Comment:
Should `ProgramData` in this case be a reference to the variable `%ProgramData%` in case the computer has the program data directory configured in a different location?
```suggestion
      <Publish Dialog="ServiceCredDlg" Control="Next" Property="JENKINS_ROOT" Value="C:\ProgramData\Jenkins\" Order="1"><![CDATA[SERVICE_LOGON_TYPE="ServiceLocalSystem"]]></Publish>
```